homework_weekend5/src/ba/bitcamp/homework_weekend5/task1/StoryCharacter.java

```diff
@@ -0,0 +1,86 @@
+package ba.bitcamp.homework_weekend5.task1;
+
+/**
+ * Describes a story character
+ * @author ajla
+ *
+ */
+public class StoryCharacter extends StoryElement {
+	private boolean sex;
+	private boolean isAlive;
+	private boolean isCapable;
+	/**
+	 * @param name
+	 * @param sex
+	 * @param isAlive
+	 * @param isCapable
+	 */
+	public StoryCharacter(String name, boolean sex, boolean isAlive,
+			boolean isCapable) {
+		super(name);
+		this.sex = sex;
+		this.isAlive = isAlive;
+		this.isCapable = isCapable;
+	}
+	
+	/**
+	 * @param name
+	 * @param sex
+	 * @param isCapable
+	 */
+	public StoryCharacter(String name, boolean sex, boolean isCapable) {
+		this(name, sex, true, isCapable);
+	}
+
+	/**
+	 * @return the isAlive
+	 */
+	public boolean isAlive() {
+		return isAlive;
+	}
+
+	/**
+	 * @param isAlive the isAlive to set
+	 */
+	public void setAlive(boolean isAlive) {
+		this.isAlive = isAlive;
+	}
+	
+	/**
+	 * @return the isCapable
+	 */
+	public boolean isCapable() {
+		return isCapable;
+	}
+
+	/**
+	 * @param isCapable the isCapable to set
+	 */
+	public void setCapable(boolean isCapable) {
+		this.isCapable = isCapable;
+	}
+
+	/**
+	 * Kills StoryCharacter on demand,
+	 * by setting isAlive property to false
+	 */
+	public void killCharacter() {
```

Comment:
Methods that perform action based on internal class state should always return boolean. True should be returned in case action was a success, else false. State that action depends on should be described in the java doc.